### PR TITLE
add modal

### PR DESCRIPTION
**Pull request by harshkotkar5557**

### CODE REVIEW

File: modals/modals.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+.modal {
+    box-shadow: #00000059 0px 5px 15px;
+    border: 1px solid var(-lynxWhite);
+    background-color: var(--lynxWhite);
```
Comment (vaibhav18matere) [inline modals/modals.css:4]: try to use color varibles instead of direct colors

File: modals/modals.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    margin: 1rem auto;
+  }
+
+ .modal p{
+   font-size: 0.8rem;
+ }
+
+.dialog-btn {
+    display: flex;
+    justify-content: flex-end;
```
Comment (vaibhav18matere) [inline modals/modals.css:20]: can make separate class for this

File: modals/modals.css [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.modal {
+    box-shadow: #00000059 0px 5px 15px;
+    border: 1px solid var(-lynxWhite);
+    background-color: var(--lynxWhite);
+    border-radius: var(--border-sm);
+    width: fit-content;
+    padding: 1rem 2rem;
+    display: flex;
+    flex-direction: column;
```
Comment (vaibhav18matere) [inline modals/modals.css:9]: can make separate class for this

**Review by vaibhav18matere — APPROVED** [review]

**Review by aayushjain-code — APPROVED** [review]
Looks good go for merge!

File: modals/modals.css (hunk 3 above)
Comment (harshkotkar5557) [reply modals/modals.css:9]: Yess I have note down all point. do it in final refactor

**Pull request merged**